site/en/codelabs/openthread-apis/index.lab.md

```diff
@@ -1025,7 +1030,7 @@ application.
 Now add some flags to the NordicSemiconductor `CMakeLists.txt`, to ensure GPIO functions are
 defined in the application.
 
-**ACTION: Add flags to the CMakeLists.txt**
+**ACTION: Add flags to the** `CMakeLists.txt` **file.**
```

Comment:
@j-noel-longhand Does this look okay on the DevSite side?

Reply:
I think so. I broke it up like this in order to prevent the line from breaking before the `code`-formatted word.